client/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -2,22 +2,19 @@ import Button from '@material-ui/core/Button';
 import Box from '@material-ui/core/Box';
 import { Link } from 'react-router-dom';
 import useStyles from './useStyles';
-import { Typography } from '@material-ui/core';
 
 interface Props {
   linkTo: string;
-  asideText: string;
   btnText: string;
 }
 
-const AuthHeader = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
+const AuthHeader = ({ linkTo, btnText }: Props): JSX.Element => {
   const classes = useStyles();
 
   return (
     <Box p={1} className={classes.authHeader}>
-      <Typography className={classes.accAside}>{asideText}</Typography>
       <Link to={linkTo} className={classes.link}>
```

Comment:
You don't need to wrap your `<Button>` in this, instead you should pass the `<Link>` with the `component` prop of the `<Button>` component. [Here is an example of how to integrate `react-router-dom` links (or other libraries) with Material-UI](https://material-ui.com/guides/composition/#button). Once you do this, you probably won't need `classes.link` as well.